### PR TITLE
Add Apple's remote desktop artifacts

### PR DESCRIPTION
**Pull request by Karneades**

### CODE REVIEW

**Comment by Karneades** [comment]
Don't know how to fix `ARNING:root:Missing symbolic link: /var/db/RemoteManagement/ClientCaches/* for path: /private/var/db/RemoteManagement/ClientCaches/* defined by artifact definition: MacOSRemoteDesktopAdministratorSystem in file: data\macos.yaml`

**Comment by joachimmetz** [comment]
@Karneades sry for the slower response can you explain more about in which context you are running into the warning. Also the validator test is not passing.

**Comment by joachimmetz** [comment]
I see where you get the warning.

```
PYTHONPATH=. ./tools/validator.py data/macos.yaml 
Validating: data/macos.yaml
WARNING:root:Missing symbolic link: /var/db/RemoteManagement/ClientCaches/* for path: /private/var/db/RemoteManagement/ClientCaches/* defined by artifact definition: MacOSRemoteDesktopAdministratorSystem in file: data/macos.yaml
WARNING:root:Missing symbolic link: /var/db/RemoteManagement/RMDB/rmdb.sqlite3 for path: /private/var/db/RemoteManagement/RMDB/rmdb.sqlite3 defined by artifact definition: MacOSRemoteDesktopAdministratorSystem in file: data/macos.yaml
WARNING:root:Missing symbolic link: /var/db/RemoteManagement/caches/AppUsage.plist for path: /private/var/db/RemoteManagement/caches/AppUsage.plist defined by artifact definition: MacOSRemoteDesktopClientSystem in file: data/macos.yaml
WARNING:root:Missing symbolic link: /var/db/RemoteManagement/caches/UserAcct.tmp for path: /private/var/db/RemoteManagement/caches/UserAcct.tmp defined by artifact definition: MacOSRemoteDesktopClientSystem in file: data/macos.yaml
FAILURE
```

File: data/macos.yaml [hunk 1]
```diff
@@ -629,6 +629,36 @@ urls:
 - 'http://forensicswiki.org/wiki/Mac_OS_X'
 - 'http://forensicswiki.org/wiki/Mac_OS_X_10.9_-_Artifacts_Location#Recent_Items'
 ---
+name: MacOSRemoteDesktopAdministratorSystem
+doc: Apple Remote Desktop (ARD) was first released in 2002 and is Apple’s desktop management system for software distribution, asset management, and remote assistance.
+sources:
+- type: FILE
+  attributes:
+    paths:
+    - '/private/var/db/RemoteManagement/ClientCaches/*'
```
Comment (joachimmetz) [inline data/macos.yaml]: Add `'/var/db/RemoteManagement/ClientCaches/*'`, etc. for completeness.

Comment (Karneades) [reply data/macos.yaml]: Add those missing symbolic link representations.

**Pull request merged**